### DISCUSSION TITLE
Auditd Manager Integration

### DISCUSSION THREAD

**Discussion by jdonovan-itcnp** [Security-Onion-Solutions/securityonion#12967] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

24

### Storage for /

100

### Storage for /nsm

60

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Migrating from 2.3 to 2.4, and at the point of configuring Elastic Agent policies in Fleet. 

Up to now, we have used AuditBeat to gather Linux audit data. There are a number of advantages to readability and useability of the audit data using Auditbeat rather than simply reading logs directly from the log files with Filebeat. With Elastic Agent, there are two different integrations available for reading audit data: 

https://www.elastic.co/guide/en/fleet/current/migrate-auditbeat-to-agent.html

Currently, SO 2.4 only supports "Auditd Logs", which reads logs from the files, similar to Filebeat. An AuditBeat replacement does exist, "Auditd Manager", however  it is not currently supported in 2.4. Technically Auditd Logs meets the need, but there are some definite disadvantages. 

So, two questions: 

1. Is it possible to implement Auditd Manager even though it is not currently officially supported? If so, how can this be done? (Obviously I will have to figure out any technical issues with implementation, just need to know how it can be allowed for access in Fleet manager.) 
2. Is it possible that Auditd Manager will be supported in future releases? Any current plans? Or, is there something which is preventing this integration from being supported. 

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
We look at adding additional integrations as time allows
Developers are staying busy with the release of Detections -> Check out the sneak peek https://www.youtube.com/watch?v=oxR4q53N6OI
